missingness/instructions.md

```diff
@@ -0,0 +1,14 @@
+# Instructions for updating the missingness analysis
+
+You should have received the following files via email:
+- `missingness.Rmd`
+- `Data_dictionary_Survey375147.csv`
+- `survey_375147_R_syntax_file.R`
+
+Download the latest version of `survey_375147_R_data_file.csv` from the WHO by exporting the 
+survey responses in the **R (data file)** format. **Put all 4 of these files in the same directory.**
+
+Open RStudio. You'll need to install 2 packages, `tidyverse` and `flextable`, before 
```

Comment:
Add a comement that rstudio might prompt her to also install other packages (pandoc, markdown, rmarkdown, etc.). Just say yes to these...

Reply:
done!